src/cache/cache_las.c

```diff
@@ -736,19 +656,19 @@ __wt_las_insert_updates(WT_CURSOR *cursor, WT_BTREE *btree, WT_PAGE *page, WT_MU
         las_key_saved = false;
 
         /*
-         * The most recent update in the list never make into WT history. Only the earlier updates
-         * in the list will make into WT history. All the entries in the WT history must have a stop
-         * timestamp.
-         */
-        stop_ts = WT_TS_MAX;
-        stop_txnid = WT_TXN_MAX;
-
-        /*
-         * Walk the list of updates, storing each key/value pair into the lookaside table. Skip
-         * aborted items (there's no point to restoring them), and assert we never see a reserved
-         * item.
+         * The algorithm assumes the oldest update on the update chain in memory is either a full
```

Comment:
I feel it would be good add an example in explaining how it traverses with scenarios where there is an update and there is an update with stop time pair on disk Or there is no on-disk value.

Reply:
I don't quite understand your comment. At this stage, we are not dealing with on-disk value. Basically, we have a value selected from the update chain to be written to the disk and the updates on the update chain.